charts/mattermost-team-edition/values.yaml

```diff
@@ -195,6 +195,9 @@ extraPorts: []
   #    port: 8585
   #    protocol: TCP
 
+# Array of extra objects to deploy with the release
+extraDeploy: []
```

Comment:
I think it is not necessary. Most of popular helm charts has this value. With extraDeploy in case of mattermost i deployed:

- additional loadBalancer for calls plugin
- added external secret with CA for postgresql SSL mode (sslmode=verify-full)

Reply:
This PR helps for

- https://github.com/mattermost/mattermost-helm/issues/392
- https://github.com/mattermost/mattermost-helm/issues/384

Reply:
> additional loadBalancer for calls plugin

This is a valid point especially for the team edition. For enterprise we recommend the scalable rtcd path in kubernetes.

> added external secret with CA for postgresql SSL mode (sslmode=verify-full)

Most of the popular charts out there what they do is to add a section `extraEnv` or `extraEnvVars`. Of course this doesn't deploy the secret resource.  So you actually add env var with reference to the secret key which can be created with a different way.

As I understand you want to couple the secret resource creation along with this helm chart vs create it independently right?

Reply:
yes